internal/pkg/cli/storage.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"github.com/aws/amazon-ecs-cli-v2/cmd/copilot/template"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/group"
+	"github.com/spf13/cobra"
+)
+
+// BuildStorageCmd is the top level command for storage
+func BuildStorageCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Hidden: true,
+		Use:    "storage",
+		Short:  "Persistent storage commands.",
```

Comment:
Can we rename `Short` and `Long` fields to be:
```
`Commands for working with objects and databases.
Augment your services with S3 buckets, NoSQL and SQL databases.`

```

I think that would look nice with the new help menu layout, https://github.com/aws/amazon-ecs-cli-v2/pull/953, what do ya think?